### DISCUSSION TITLE
A better name for this game

### DISCUSSION THREAD

**Discussion by pserwylo** [beat-feet/beat-feet#40] [Ideas]
I used "beat-game" as an in-progress name for myself. It could probably be improved. Happy to entertain any ideas. Once we settle on a new name, then I'd actually like to transfer this repository to an organisation of the same name. So, some quick heuristics:

* A quick google search (particularly in the play store) should not turn up any similar sounding apps.
* A preference for the start of the English alphabet (selfishly because it will appear near the top of F-Droid when a new version is released ;)
* Although I love a good pun as much as the next person, it should be something that is not too difficult to understand for non-native English speakers.

**Comment by sanctimoniousape** [comment]
"Beat Feet" or "Beat Feat"

**Reply by pserwylo** [reply]
Ooh, I like this. I especially like the Feat version, but it will probably internationalise better with Beat Feet. I'm leaning towards renaming to this, but will wait another week or so in case there are any others that jump out on this thread.

**Reply by pserwylo** [reply]
Alright, thanks again for the suggestion.

I've renamed the game to "Beat Feet", including renaming this repository, the GitHub organisation, Liberapay account, etc. Will bump a new v0.8.0 so hopefully F-Droid picks up the rename correctly.

**Comment by PNX-Records** [comment]
The game reminds me of the sport [Parkour](https://en.m.wikipedia.org/wiki/Parkour)  where people use urban landscapes as hurdle track. Since the game is driven by music I'd like to recommend as a name
"Musical Parkour" or if you feel the beat aspect important "Beat Parkour".

**Reply by pserwylo** [reply]
Thanks so much for participating. I think it is definitely sensible, but compared to Beat Feet it is just that little bit longer. 

Happy to take votes from others in the community though for a while.
